Project.toml

```diff
@@ -6,28 +6,28 @@ version = "0.2.3"
 [deps]
 AutoHashEquals = "15f4f7f2-30c1-5605-9d31-71845cf9641f"
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
-ConditionalJuMP = "ae04f764-fc8b-5ee0-af1c-aa760b5c9409"
```

Comment:
We vendor the functions used in `ConditionalJuMP`, removing this as a dependency (since it did not support JuMP >= 0.18) and adding `IntervalArithmetic` instead,